### DISCUSSION TITLE
Coder-server running in background issue

### DISCUSSION THREAD

**Discussion by wabbyy** [coder/code-server#3033] [General]
Every time I connect to control panel to start coder-server, but if it disconnected, then I can not log through IP, to use coder-server.

This is how I start coder-server：
export PASSWORD="passward" && nohup ./code-server --host 0.0.0.0 --port 80 &

Ubuntu: 20.04.1
Coder-server:  3.9.2
Cloud server: Alibaba Could (ESC)

Can someone help with me？

**Comment by code-asher** [comment]
I haven't used `nohup` before but maybe it's not preventing code-server from getting killed when the session disconnects? Some ideas:

1. Use screen or tmux
2. Use a systemd service (if you install with our install script this will be created for you)
